src/validation/water_uptake/CMakeLists.txt

```diff
@@ -39,8 +40,10 @@ foreach (input
           modal_aero_water_uptake_wetaer
           modal_aero_water_uptake_dryaer
           modal_aero_water_uptake_dr
-          modal_aero_water_uptake_dr_wetdens
+          # FIXME: fix this test
```

Comment:
Should this be fixed before the PR goes in, or is this something you'd like to save for later?

Reply:
It is fixed.